.gitlab-ci.yml

```diff
@@ -2,7 +2,7 @@ variables:
   # These are gitlab variables so that it's easier to do a manual deploy
   # If these are set witih value and description, then it gives you UI elements
   DOWNSTREAM_BRANCH:
-    value: "main"
+    value: "levi/rust-1.64"
```

Comment:
should this be changed back to main?

Reply:
Yes, but not until that downstream PR is merged.